src/components/molecules/CardInput/CardInput.tsx

```diff
@@ -1,19 +1,7 @@
-import React from "react";
-import { CardElement } from "@stripe/react-stripe-js";
 import "./CardInput.scss";
 
-const CARD_ELEMENT_OPTIONS = {
-  style: {
-    base: {
-      color: "white",
-      fontSmoothing: "antialiased",
-      fontSize: "16px",
-    },
-  },
-};
-
 const CardInput = () => {
-  return <CardElement options={CARD_ELEMENT_OPTIONS} />;
+  return;
 };
```

Comment:
Why don't you just delete the whole component?

Reply:
fix in f771f8b9a7f56d7ffc149fa83c3a794f916870c6